data/darktableconfig.xml.in

```diff
@@ -2124,6 +2124,19 @@
     <shortdescription>border around image in darkroom mode</shortdescription>
     <longdescription>process the image in darkroom mode with a small border. set to 0 if you don't want any border.</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>darkroom/ui/iso12464_border</name>
+    <type>float</type>
+    <default>4.0</default>
+    <shortdescription>total width of border in iso 12646 mode (cm)</shortdescription>
```

Comment:
so default is 4.0 cm, but that's not what I'm seeing.

I think this is due to:

`tb = MIN(MAX(2, bsize), 0.3f * MIN(wd, ht));`

Reply:
That just ensures we have at least a total border size of 2 but will never be greater than 30% of width or height.